collector/sbd/sbd.go

```diff
@@ -72,14 +71,13 @@ func (c *sbdCollector) CollectWithError(ch chan<- prometheus.Metric) error {
 		ch <- c.MakeGaugeMetric("devices", 1, sbdDev, sbdStatus)
 	}
 
-	sbdWatchdogs := c.getSbdWatchDogTimeout(sbdDevices)
+	sbdWatchdogs, sbdMsgWaits := c.getSbdTimeouts(sbdDevices)
 	for sbdDev, sbdWatchdog := range sbdWatchdogs {
-		ch <- c.MakeGaugeMetric("watchdog_timeout", sbdWatchdog, sbdDev)
+		ch <- c.MakeGaugeMetric("sbd_timeouts", sbdWatchdog, sbdDev, "watchdog_timeout")
```

Comment:
I think we can avoid these repetitions:
```suggestion
		ch <- c.MakeGaugeMetric("timeouts", sbdWatchdog, sbdDev, "watchdog")
```